### ISSUE TITLE
Allow editing during routing

### ISSUE THREAD

**Issue by rugk** [mapsme/omim#4507]
Currently during routing one cannot edit the map. This is however bad for mappers, who want to get to a place and edit the map during travelling there.
I think this is not a such uncommon situation (at least when you walk or go by bike).

Therefore I suggest you to change this.

**Comment by dbf256** [comment]
I've already reported it to bugs@maps.me and got a reply that it is not implemented because of the difficulties to do it in 'perspective view' in navigation mode. Still don't understand why it is not possible to do it in 2D view.

**Comment by rugk** [comment]
Yes, I also rather mean editing POIs, which are already on the map rather than adding new entries. I think at least this should be possible.

**Comment by burivuh** [comment]
I think it can be possible for a pedestrian routing mode.
We will not do it in any vehicle mode though due to the safety issues.

**Comment by dbf256** [comment]
I do some mapping during bicycle walks (in safe mode!), but if you add at least for pedestrian mode it will be great.
